codebase/superdarn/src.idl/lib/main.1.25/cnvmap.pro

```diff
@@ -228,7 +228,7 @@ function CnvMapRead,unit,prm,stvec,gvec,mvec,coef,bvec
            
 
   arrtype=[2,2,2,4,2,2,2,4,4,2,4,4,4,4,4,4,4,4, $
-           4,4,4,2,2,3,4,4,4,4,4,4,8,8,8,8,4,4,4,4,4,4]
+           4,4,4,4,2,2,3,4,4,4,4,4,4,8,8,8,8,4,4,4,4,4,4]
```

Comment:
Would be useful to have a comment here shortly saying what this array is holding.

Reply:
I think we'll leave this one for resolving in #631